src/ext_slice.rs

```diff
@@ -1324,6 +1324,9 @@ pub trait ByteSlice: Sealed {
     /// let x: Vec<_> = b"abcXdef".splitn_str(1, "X").collect();
     /// assert_eq!(x, vec![B("abcXdef")]);
     ///
+    /// let x: Vec<_> = b"abcdef".splitn_str(2, "X").collect();
+    /// assert_eq!(x, vec![B("abcdef")]);
+    ///
```

Comment:
Could you add a corresponding test case to `rsplitn_str` as well?

Reply:
Sure, done :) Thanks for the fast response!